src/GitVersionTask.MsBuild/GitVersionTask.MsBuild.csproj

```diff
@@ -6,8 +6,12 @@
         <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     </PropertyGroup>
 
+    <PropertyGroup Condition="'$(TargetFramework)' == 'netstandard2.0' ">
+        <DefineConstants>UseSystemRuntimeLoader</DefineConstants>
+    </PropertyGroup>
+
     <ItemGroup>
-        <PackageReference Include="System.Runtime.Loader" Version="4.3.0" />
+        <PackageReference Condition="$(DefineConstants.Contains(UseSystemRuntimeLoader))" Include="System.Runtime.Loader" Version="4.3.0" PrivateAssets="All" />
```

Comment:
I wonder if you can use directly the Targetframework property instead of creating a new one

Reply:
I can do, this was personal preference as I prefer to keep condition expressions simple. In this case there is only one target framework involved so my approach isn't really paying much dividends. I'll change it.